.bcr/metadata.template.json

```diff
@@ -0,0 +1,20 @@
+{
+  "homepage": "https://github.com/engflow/gazelle_cc",
+  "maintainers": [
+    {
+      "name": "Jay Conrod",
+      "email": "jayconrod@gmail.com",
```

Comment:
Question for @jayconrod: Do you want your personal email here, or your EngFlow email?